### ISSUE TITLE
Build instructions are incomplete.

### ISSUE THREAD

**Issue by synercoder** [emgucv/emgucv#103]
TL;DR:
---
Add "Desktop development with C++" as a prerequisite in the instructions.

Long version:
---
I cloned the repo and followed these instructions: http://www.emgu.com/wiki/index.php/Download_And_Installation#Building_from_Git . The prerequisites are listed as:

- You will need to install CMAKE in-order to build the unmanaged C++ code (OpenCV and cvextern.dll).
- You will need Visual Studio 2015/ Visual Studio 2013 to build the Managed code (Emgu CV)

But in order to run CMAKE, you also need to have Visual Studio installed with "Desktop development with C++" workload enabled. I am a purely C# developer, and I got CMAKE errors about VCTargets or something. I looked into the missing file, and it was a ???.cpp.???.props file. Therefor I installed the "Desktop development with C++" workload

![image](https://user-images.githubusercontent.com/18616295/36843044-f278509a-1d4d-11e8-9303-2a18360747ac.png)

Afterwards, I could run CMAKE correctly on the project.

**Comment by emgucv** [comment]
Good point — the C++ build tools ("Desktop development with C++" workload in Visual Studio, or the standalone Build Tools for Visual Studio) are required for CMake to generate and build the native `cvextern` library. The repository's `CLAUDE.md` now contains up-to-date build instructions that cover the full prerequisites. Closing now.
